addons/controllers/csi/vspherecsiconfig_utils.go

```diff
@@ -26,6 +30,56 @@ import (
 	topologyv1alpha1 "github.com/vmware-tanzu/vm-operator/external/tanzu-topology/api/v1alpha1"
 )
 
+// ClusterToVSphereCSIConfig returns a list of Requests with VSphereCSIConfig ObjectKey
+func (r *VSphereCSIConfigReconciler) ClusterToVSphereCSIConfig(o client.Object) []ctrl.Request {
```

Comment:
Hi @lubronzhan, could you help explain why we queue event based on `vSphereCluster` for vsphere CPI config controller, but based on `Cluster` for CSI controller?

Reply:
I just changed it to watching vSphereCluster instead